client/core/trade.go

```diff
@@ -1950,6 +1954,13 @@ func (t *trackedTrade) auditContract(match *matchTracker, coinID []byte, contrac
 				tries++
 				return wait.TryAgain
 			}
+			// Even retry for unrecognized errors, at least for a little while.
+			// With a default recheckInterval of 5 seconds, this is 2 minutes.
+			if tries < 24 {
```

Comment:
@buck54321 I made a change to have it go for a flat 2 minutes, but (a) that makes the unit tests go for 2 minutes where it expects audit to fail in certain cases and (b) unit tests are using a crazy short recheck interval so it's retries like 50 million times.  I just revised the comment instead.